### DISCUSSION TITLE
Info on the branches of Crappy

### DISCUSSION THREAD

**Discussion by WeisLeDocto** [LaboratoireMecaniqueLille/crappy#30] [Q&A] [answered]
Dear @Dad0u,

Do you remember why the branches **1.2** and **py2** were created and kept until now ?
To your knowledge, are they still in use to any users ?

Thank you,
Weis

**Comment by Dad0u** [comment, marked as answer]
As far as i can remember, Biotens was the last of our setups to use Crappy 1.2 but i believe it has been updated since. This branch can probably be archived now.

The py2 branch was used to provide support for Python 2, which was still used by the braking tribometer. There was some ongoing development to move the braking tribometer to Python 3 but i do not know if this is done.

However, i cannot tell if anyone else is using either of this branches.

What we can tell is that there is no active dev on these branches anymore so i would suggest to archive them.
Hope this helps !

**Reply by WeisLeDocto** [reply]
Got it, thank you !
